tkcalendar.py

```diff
@@ -104,6 +105,14 @@ def __init__(self, master=None, **kw):
         # --- date
         today = self.date.today()
 
+        try:
```

Comment:
No need to use a try/except block here, instead use `pop(key, fallback)` to avoid raising the `KeyError`. In addition, I think it would be simpler to create a `StringVar` to avoid testing its existence before changing its content later in the code.